examples/bpk-component-bottom-sheet/examples.tsx

```diff
@@ -0,0 +1,195 @@
+    Aenean tempus tempus lorem in consequat. Quisque nec felis vitae mi commodo
+    ultricies sit amet in lectus. Praesent imperdiet auctor nisl sit amet
+    ultricies. Donec posuere placerat nulla a scelerisque. Nulla sit amet
+    eleifend magna. Ut eu cursus metus, id pulvinar lectus. Proin euismod sed ex
+    vel dignissim. Donec faucibus nec risus eu pellentesque. Cras vulputate
+    varius volutpat. Duis ut nisi nulla. Duis volutpat lectus purus. Aliquam
+    placerat dignissim mauris vitae interdum. Donec venenatis mattis mi eu
+    facilisis. Maecenas pellentesque eros elementum, tincidunt tortor ac,
+    fringilla massa. Cras sed orci nec erat porttitor lacinia vitae sed arcu.
+  </Paragraph>,
+  <Paragraph>
+    Nunc lobortis arcu eget tellus tincidunt commodo. Phasellus ac tortor
+    turpis. Cras ac quam non metus iaculis sollicitudin. Vestibulum ante ipsum
+    primis in faucibus orci luctus et ultrices posuere cubilia Curae; Morbi
+    aliquam consectetur nisl at fermentum. Phasellus nisi arcu, fermentum ut
+    malesuada eu, ultrices nec enim. Donec eleifend eros mauris, eu rutrum
+    tellus suscipit ac. Pellentesque finibus mollis arcu, non tempor lorem
+    gravida at. Nam laoreet neque quis gravida blandit. Mauris pharetra urna
+    hendrerit pretium auctor. Aliquam erat volutpat. Class aptent taciti
+    sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos.
+    Fusce quis felis non lectus egestas pretium id sed erat. Aliquam dapibus
+    erat sit amet facilisis luctus.
+  </Paragraph>,
+  <Paragraph>
+    Vestibulum convallis ut nulla in iaculis. Aliquam erat volutpat. Nullam non
+    semper sem. Ut gravida est eu nisi condimentum, lobortis gravida ipsum
+    tincidunt. Duis lacinia tincidunt suscipit. Maecenas tincidunt quam ipsum,
+    non sodales ante placerat in. Suspendisse malesuada auctor erat, vel
+    pulvinar erat dignissim vitae.
+  </Paragraph>,
+];
+
+class BottomSheetContainer extends Component<
+  {
+    children: ReactNode,
+  },
+  {
+    isOpen: boolean,
+  }
+> {
+  static propTypes = {
+    children: PropTypes.node.isRequired,
+  };
+
+  constructor() {
+    super();
+
+    this.state = {
+      isOpen: false,
+    };
+  }
+
+  onOpen = () => {
+    this.setState({
+      isOpen: true,
+    });
+  };
+
+  onClose = () => {
+    this.setState({
+      isOpen: false,
+    });
+  };
+
+  render() {
+    const { children, ...rest } =
+      this.props;
+
+    return (
+      <div id="modal-container">
+        <div id="pagewrap">
+          <BpkButton onClick={this.onOpen}>
+            Open bottom sheet
+          </BpkButton>
+          {/* $FlowFixMe[cannot-spread-inexact] - inexact rest. See 'decisions/flowfixme.md'. */}
+          <BpkBottomSheet
+            id="my-modal"
+            className="my-classname"
+            isOpen={this.state.isOpen}
+            onClose={this.onClose}
+            getApplicationElement={() => document.getElementById('pagewrap')}
+            {...rest}
```

Comment:
Do we need to provide the `renderTarget` here so that it can attach correctly to the container?